README.md

```diff
@@ -1 +1,27 @@
 # Istio 101
+
+This is the repository for the [Istio][istio] 101 workshop from IBM.
+
+- [presentation](./presentation) -- the data for the teacher/leader of the workshop
+- [workshop](./workshop) -- the vanilla code for the code of the workshop
+
+
+```text
+Copyright:: IBM, Inc 2018-
```

Comment:
is "Copyright::" supposed to show up as (C) or something?

Reply:
Nope, it's just a way to say `Copyright::`.